netci.groovy

```diff
@@ -252,6 +252,21 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
                     }
                 }
                 break
+            case 'r2r':
+                //r2r jobs that aren't pri1 can only be triggered by phrase
+                break
+            case 'pri1r2r':
+                //pri1 r2r gets a push trigger for checked/release
+                if (architecture == 'x64' || architecture == 'x86') {
```

Comment:
We want for both. I was simply saying that for x86, per-commit should only happen for Windows OS and not any other OS.